controllers/front/CartController.php

```diff
@@ -653,15 +674,35 @@ private function checkCartProductsMinimalQuantities()
         foreach ($productList as $product) {
             if ($product['minimal_quantity'] > $product['cart_quantity']) {
                 // display minimal quantity warning error message
-                $this->errors[] = $this->trans(
-                    'The minimum purchase order quantity for the product %product% is %quantity%.',
-                    [
-                        '%product%' => $product['name'],
-                        '%quantity%' => $product['minimal_quantity'],
-                    ],
-                    'Shop.Notifications.Error'
+                $this->errors[] = $this->translateProductNameWithAttributes(
+                    'The minimum purchase order quantity for the product [1]%product%[/1] is [1]%quantity%[/1].',
```

Comment:
```suggestion
                    'The minimum order quantity for the product [1]%product%[/1] is [1]%quantity%[/1].',
```

Reply:
Couldn't we just say "this product" like in the other PR? https://github.com/PrestaShop/PrestaShop/pull/29016/files

Reply:
Sure 😊